idb/postgres/internal/writer/writer.go

```diff
@@ -339,6 +377,13 @@ func (w *Writer) AddBlock(block *bookkeeping.Block, modifiedTxns []transactions.
 		}
 		writeAccountDeltas(block.Round(), &delta.Accts, sigTypeDeltas, &batch)
 	}
+	{
+		// TODO: don't think I actually need this code block
```

Comment:
^^^ figure out ^^^

Reply:
Hard for me to parse your internal monologue.  Do you mean you're trying to decide if you need the indentation and extra scope? Or whether you need the code at all? It _seems_ like this is the heart of the matter here - the code is needed - so I think you're debating whether to introduce the scope?  If so, I'd say do it, since the same technique was used above.  (Or remove both if this the only place this is done) No point in change style mid-stream.

I assume it was done to isolate the introduction of this `err` variable?